pkg/auth/auth_handler.go

```diff
@@ -219,6 +220,8 @@ func (a *AuthHandler) CreateAuthenticator() func(name, in string, authenticate s
 			}
 			p, err := authenticate(token)
 			if err != nil {
+				log := logutil.FromContext(r.Context(), a.log)
```

Comment:
probably should move the beginning of the function then if we want to add some logs we will be able to do so.

Reply:
Done